app/services/proofing/resolution/progressive_proofer.rb

```diff
@@ -202,6 +201,22 @@ def threatmetrix_disabled_result
         )
       end
 
+      def threatmetrix_pii_missing_result
+        Proofing::DdpResult.new(
+          success: false,
+          client: 'tmx_pii_missing',
```

Comment:
Note to other reviewers: abusing the `client` field for these kind of statuses is not ideal, but it is a pattern that is established in the codebase, but also not something we're looking to solve in this PR.